### PR TITLE
update readme links

### PR DESCRIPTION
**Pull request by asuncepeda**
Add PR ids to the links and expand a little  the steps about check temporary branches

### CODE REVIEW

File: iterate_snowflake/README.md [hunk 1]
```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
 - Create a new git branch 
 - Create a new Snowflake .datasource data file in the branch, as in the example PR. You shouldn't need fixtures as the Data Source will be populated from the connection, but you can add fixtures and/or tests if needed.
   - For information about Snowflake options read the [connector documentation](https://www.tinybird.co/docs/ingest/snowflake)
-- Commit your code and create a new PR/MR from the branch to `main`
-- Wait for CI. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Commit your code and create a new PR/MR from the branch to `main` 
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
```
Comment (alrocar) [inline iterate_snowflake/README.md:13]: Could we be more specific? How do I check that everything is working, do I need to `sync now` the data source?

File: iterate_snowflake/README.md [hunk 2]
```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
...
 
 
 ### Update Snowflake Data Source with a new column, with same connection
 
 - Create another git branch
 - Add the new column to the schema and query
 - Commit your code and create an ew PR from the branch to `main`
-- Wait for CI to finish. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
```
Comment (alrocar) [inline iterate_snowflake/README.md:25]: Same here. How do we check that it works? Does the connector works when it was previously created?

File: iterate_snowflake/README.md [hunk 3]
```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
...
 - Merge and wait for CD. Now you should have your Data Source updated in the main workspace, using the connection, and the temporary CI branch will be deleted.
 
-[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/tbd)
+[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/360)
 
 ### Delete a Snowflake Data Source
 
 - Create another git branch
 - Remove the .datasource data file
 - Commit your code and create an ew PR from the branch to `main`
```
Comment (alrocar) [inline iterate_snowflake/README.md:34]:
```suggestion
- Commit your code and create a new PR from the branch to `main`
```

Same typo in other places 🙏

File: iterate_snowflake/README.md [hunk 4]
```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
...
-- Wait for CI to finish. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
 - Merge and wait for CD. Now you should have your Data Source updated in the main workspace, using the connection, and the temporary CI branch will be deleted.
```
Comment (alrocar) [inline iterate_snowflake/README.md:37]: `Now you should have your Data Source updated in the main workspace`

The DS is deleted right? (only if it did not have dependencies I guess)

What about the connection is it still there?

**Review by alrocar — COMMENTED** [review]
Some comments, the more specific we can be the better b/c at some point someone will read it and won't know how to proceed or to expect 🙏

File: iterate_snowflake/README.md (hunk 2 above)
Comment (asuncepeda) [reply iterate_snowflake/README.md:38]: Same that above

File: iterate_snowflake/README.md (hunk 4 above)
Comment (asuncepeda) [reply iterate_snowflake/README.md]: Added a note about dependencies an about the connection.

File: iterate_snowflake/README.md (hunk 2 above)
Comment (alrocar) [reply iterate_snowflake/README.md:38]: are we 100% sure about this? I thought connectors didn't work in branches (except if you create them for the first time), so I'd expect the connector not to work if you modify the data source, not sure if I'm missing anything 🙏

Comment (asuncepeda) [reply iterate_snowflake/README.md:38]: I've made a ton of tests, but going to make another 😅

Comment (asuncepeda) [reply iterate_snowflake/README.md:38]: I've left a couple of tests over https://app.tinybird.co/gcp/europe-west3/asun_20240909/datasource/created_no_fixture
* When you create a tb branch directly (in this case `from_ui_connection_dont_work` the connection doesn't works (surprise!), idem if you create the branch with cli, etc
* But when you run CI (branch `tmp_ci__5`) the connection is OK, I understand this happens b/c not only the branch is created, but then the changes in the datasource are applied.
In fact it's the same we're telling the users to do, if you create a branch the connector breaks and you must re-create the ds. I've tested that in the branch `from_cli_to_push`:
* I've create the branch from main and checked that the DS didn't work
* Then I've manually `tb push -f` the data source and now it works

Comment (alrocar) [reply iterate_snowflake/README.md:38]: Perfect then, thanks!

**Review by alrocar — APPROVED** [review]

**Pull request merged**